crates/bevy_render/src/view/window.rs

```diff
@@ -167,9 +167,14 @@ pub fn prepare_windows(
             height: window.physical_height,
             usage: wgpu::TextureUsages::RENDER_ATTACHMENT,
             present_mode: match window.present_mode {
+                #[allow(deprecated)]
                 PresentMode::Fifo => wgpu::PresentMode::Fifo,
+                #[allow(deprecated)]
                 PresentMode::Mailbox => wgpu::PresentMode::Mailbox,
+                #[allow(deprecated)]
                 PresentMode::Immediate => wgpu::PresentMode::Immediate,
+                PresentMode::AutoVsync => wgpu::PresentMode::AutoVsync,
```

Comment:
`AutoVsync` is a bit of an odd setup considering people choose vsync to avoid tearing, and `AutoVsync` uses `FifoRelaxed` if supported (AMD Vulkan only) and falls back to `Fifo`. `FifoRelaxed` behaves like `Fifo` for fps > vsync, and will tear for fps < vsync, which to me seems like a poor choice and not what I would want if I choose vsync.